Source/UIKit additions/SVGKLayeredImageView.m

```diff
@@ -75,28 +79,33 @@ - (void)populateFromImage:(SVGKImage*) im
 	{
 #ifndef SVGK_DONT_USE_EMPTY_IMAGE_PLACEHOLDER
         SVGKitLogWarn(@"[%@] WARNING: you have initialized an [%@] with a blank image (nil). Possibly because you're using Storyboards or NIBs which Apple won't allow us to decorate. Make sure you assign an SVGKImage to the .image property!", [self class], [self class]);
-		
+#if SVGKIT_UIKIT
 		self.backgroundColor = [UIColor clearColor];
+#else
+        self.layer.backgroundColor = [NSColor clearColor].CGColor;
+#endif
         
-        NSString* svgStringDefaultContents = @"\
```

Comment:
Moving these constant string into utils. Because now there are multiple files need this...No need to write twice. See MaddTheSane's change

Reply:
This is tightly bound to the class it is in - it's data for the class implementation.

What are the other places that are using it?

Reply:
The [MaddTheSane's SVGKImageRep](https://github.com/MaddTheSane/SVGKit/blob/master/Source/AppKit%20additions/SVGKImageRep.m#L209)'s default init use this value... 😅 

Or maybe we can just ignore the default init method ?